packages/sdk/typescript/human-protocol-sdk/src/escrow.ts

```diff
@@ -1688,6 +1814,11 @@ export class EscrowUtils {
       throw ErrorUnsupportedChainID;
     }
 
+    let statuses;
+    if (filter.status !== undefined) {
+      statuses = Array.isArray(filter.status) ? filter.status : [filter.status];
+      statuses = statuses.map((status) => EscrowStatus[status]);
+    }
```

Comment:
Isn't it already in `main`? Probably you should pull latest changes to this branch